### PR TITLE
enable dgAssembly

### PR DESCRIPTION
**Pull request by klawani**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

enable dgDocAssembly and bump chart version

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: cmc/values.preview.yaml [hunk 1]
```diff
@@ -53,6 +53,7 @@ cmc-claim-store:
       PDF_SERVICE_URL: '{{ .Values.global.pdfServiceUrl }}'
       SEND_LETTER_URL: '{{ .Values.global.sendLetterUrl }}'
       DOCUMENT_MANAGEMENT_URL: '{{ .Values.global.documentManagementUrl }}'
+      DOC_ASSEMBLY_URL: '{{ .Values.global.dgAssemblyUrl }}'
```
Comment (adusumillipraveen) [inline cmc/values.preview.yaml]: Shouldn't need this

Comment (klawani) [reply cmc/values.preview.yaml]: it using AAT endpoint ...

Comment (adusumillipraveen) [reply cmc/values.preview.yaml]: You are overriding this line with same value. https://github.com/hmcts/chart-cmc/pull/43/files#diff-483f7a4b2394c948e242ab7122a8aa52R66
```suggestion
```

Comment (klawani) [reply cmc/values.preview.yaml]: done

**Pull request merged**